### PR TITLE
Added Symphony

### PR DESCRIPTION
**Pull request by xxdjbobbyxx**
### Submissions:

* [x] Does your submission meet with our requirements?
* [x] Is the JSON file formatted correctly?
* [x] Have you checked to ensure there aren't other open [pull requests](../../../pulls) for the same update/change?

### CODE REVIEW

**Comment by DefectivenessYT** [comment]
Stream and API seem to be offline @xxdjbobbyxx, re-request review when all of this is sorted.

**Review by DefectivenessYT — CHANGES_REQUESTED** [review]
ensure api and live link function before pr'ing

**Comment by xxdjbobbyxx** [comment]
Fixed the new issues.

**Review by DefectivenessYT — CHANGES_REQUESTED** [review]
live link doesn't work?

**Comment by xxdjbobbyxx** [comment]
Port defined and mount point now created

**Review by DamonOnYT — CHANGES_REQUESTED** [review]
Links should be HTTPS.

API will also be added to RSpy meaning api_url should be "external", I will update the PR when it is.

**Comment by xxdjbobbyxx** [comment]
We dont have a SSL because its on port 90 and therefor the website wont work without it being on port 80 so feel free to remove this pull request if you do not want Symphony on

**Comment by DamonOnYT** [comment]
@xxdjbobbyxx Do you not understand how web ports work? You are fowaring everything to port 80 where Cloudflare then has SSL on their end.

**Comment by xxdjbobbyxx** [comment]
Not everything is going through cloudflare as thats what is breaking shit on my end

**Comment by DamonOnYT** [comment]
@xxdjbobbyxx Issue SSL via Azuracast. Considering the site is also only in HTTPS, We will not be able to merge it until you can sort out SSL Certificates.

I will be able to sort out SSL for the API, as its passed through RadioSpy however the radio SSL will need to be sorted by you.

**Comment by DefectivenessYT** [comment]
Your radio is now showing "service unavailable." if you fix this, i'll proxy the radio stream through me for SSL.

**Comment by xxdjbobbyxx** [comment]
> Your radio is now showing "service unavailable." if you fix this, i'll proxy the radio stream through me for SSL.

its working fine for me

**Comment by DefectivenessYT** [comment]
All URLs is now secure and functioning, waiting on @DamonOnYT to add to radiospy api, then @jackkelly-bayliss or @JackKellyBayliss will proceed to add to RadioPeeker

**Review by DefectivenessYT — APPROVED** [review]

**Comment by jackkelly-bayliss** [comment]
@DamonOnYT Let me know once this has been added to RadioSpy and I'll approve on my end.

**Comment by DefectivenessYT** [comment]
@jackkelly-bayliss symphony has been added to the radiospy api 👍

File: radios/S/symphonyradio/radio.json [hunk 1]
```diff
@@ -3,12 +3,12 @@
     "symphonyradio":
       { 
         "name": "SymphonyRadio",
-        "logo": "http://symphonyradio.tk/assets/img/logo.png",
-        "website": "http://symphonyradio.tk",
+        "logo": "https://symphonyradio.tk/assets/img/logo.png",
```
Comment (HexDevv) [inline radios/S/symphonyradio/radio.json:6]: Consider moving away from a .tk domain. People are able to 'out-buy' it afaik.

File: radios/S/symphonyradio/radio.json [hunk 2]
```diff
@@ -3,12 +3,12 @@
...
+        "website": "https://symphonyradio.tk",
```
Comment (HexDevv) [inline radios/S/symphonyradio/radio.json:7]: Consider moving away from a .tk domain. People are able to 'out-buy' it afaik.

File: radios/S/symphonyradio/radio.json [hunk 3]
```diff
@@ -3,12 +3,12 @@
...
         "description": "SymphonyRadio, bringing symphony to your ears",
         "color": "#46356e",
         "listen_url": "https://azura.bencaulfield.me/radio/8120/main.mp3,
-        "api_url": "http://api.symphonyradio.tk",
+        "api_url": "https://api.symphonyradio.tk",
```
Comment (HexDevv) [inline radios/S/symphonyradio/radio.json:11]: Consider moving away from a .tk domain. People are able to 'out-buy' it afaik.

**Review by HexDevv — CHANGES_REQUESTED** [review]
Consider moving away from a .tk domain. People are able to 'out-buy' it afaik.

**Review by DefectivenessYT — APPROVED** [review]

**Comment by xxdjbobbyxx** [comment]
I will be buying a domain within the next couple of days

**Comment by DefectivenessYT** [comment]
Perfect, for now though, this is fine, and make sure to commit a PR when you have changed domain.

**Review by DamonOnYT — CHANGES_REQUESTED** [review]
1. Listen URL is not closed properly. It is missing a " after the stream.
2. Change api_url to "external" as its on RadioSpy
3. Change Logo, Website, and find a listen url thats https.

**Review by HexDevv — APPROVED** [review]
> I will be buying a domain within the next couple of days

fine by me then. please fix Damon's issues then you'll be fine 👍

**Comment by DefectivenessYT** [comment]
HEX U DONT EVEN WORK HERE SADUYDSAGBDS

**Comment by xxdjbobbyxx** [comment]
Fixed the issues

**Review by DamonOnYT — APPROVED** [review]
Looks good. Got my approval.

**Comment by HexDevv** [comment]
> HEX U DONT EVEN WORK HERE SADUYDSAGBDS

still open source, even so, could always hire me 😉

**Comment by DefectivenessYT** [comment]
please refer to your previous issue.

**Pull request merged**